src/library/ziputils.lua

```diff
@@ -5,7 +5,6 @@ Functions for unzipping files. (Future may include zipping as well.)
 
 Dependencies:
 
-- Windows users must have `7z` installed. You can download it [here](https://www.7-zip.org/).
 - MacOS users must have `unzip` and `gunzip`, but these are usually installed with the OS.
 
```

Comment:
I went ahead and added a comment (and refactored some of the others). If I need to change it, let me know. I want to refactor this to split the `enigmaxml` support out into its own library.

Reply:
Funny, I didn't get this comment notification until after you had merged!

Some version of PowerShell has been integrated with Windows since as far back as Windows 7. I think the calls I'm making are pretty vanilla and should work with PS 3, which was included with Win 8. There's a slightly complicated way to check compatibility with earlier versions; I'll try to get to this later today.

PS 5.1, which is current, was preinstalled starting with Win 10.

Reply:
The minimum version for Finale is 8.1, if that makes a difference. It will update the comments to say that it must be manually installed for 8.1. (In the middle of significant refactoring at the moment.)

Reply:
Looks like Win 8.1 came with PowerShell 4, which is still not enough. I'm actually not sure how one would upgrade that from 4 to 5. It may have come for free with Windows Update at some point.

Reply:
Anyone still on 8.1 is extremely determined and can figure it out for themselves.